### PR TITLE
Feature poi tests

### PR DESCRIPTION
**Pull request by ali1373107**
adding poi route has been tested and test passed. 
sum.js file deleted. its not necessarily to have it.

### CODE REVIEW

File: sum.js
Comment (TomStrong) [inline sum.js:1]: sum.js file appears to still be present

File: test-esm.js
Comment (TomStrong) [inline test-esm.js:1]: Is this file necessary? What does it do?

**Review by TomStrong — CHANGES_REQUESTED** [review]
Please address the comments I have made:

- Remove sum.js
- Explain why we need test-esm.js

File: tests/routs.test.js
Comment (TomStrong) [inline tests/routs.test.js:1]: This file is incorrectly named

**Review by TomStrong — CHANGES_REQUESTED** [review]
Change file name as shown in comments

File: test-esm.js
Comment (ali1373107) [reply test-esm.js:1]: the reason i added this code to your test file is to enable the use of ES6 module syntax in your tests. but i deleted unnecessarily files. check it and let me know if any other changes required

**Review by 5thaks07 — APPROVED** [review]
Very nice mate

**Review by TomStrong — APPROVED** [review]

**Pull request merged**